lib/kernel-manager.coffee

```diff
@@ -11,6 +11,7 @@ module.exports =
 class KernelManager
     constructor: ->
         @_runningKernels = {}
+        @_kernelSpecs = []
```

Comment:
Thanks a lot for your review! I really appreciate your feedback.

> @n-riesco I think @_kernelSpecs should be initialised with the user settings; i.e. @_kernelSpecs = @parseKernelSpecSettings(). Is there a case when we wouldn't this to happen?

The parsing of user settings should happen in `updateKernelSpecs`, otherwise `hydrogen:update-kernels` wouldn't help if the settings are changed by the user.